Ports/julius/package.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+
+port="julius"
+version="1.7.0"
+useconfigure="true"
+files="https://github.com/bvschaik/julius/archive/refs/tags/v${version}.tar.gz v${version}.tar.gz 3ee62699bcbf6c74fe5a9c940c62187141422a9bd98e01747a554fd77483431f"
+depends=(
+    "cmake" 
+    "SDL2" 
+    "SDL2_mixer" 
+    "libpng"
+)
+
+configopts=(
+    "-DSYSTEM_LIBS=OFF"
```

Comment:
Are you sure this line is necessary? It looks like this determines whether an internal version of `libpng` is used or not. We want our `libpng` to be used here. 

Omitting this seems to make no difference to the build.